package.json

```diff
@@ -82,12 +82,12 @@
     },
     "devDependencies": {
         "@dhis2/cli-style": "^4.1.1",
-        "code": "^5.2.4",
+        "@hapi/joi": "^16.1.7",
```

Comment:
I think joi is supposed to go as a normal dep and not devDependency

Reply:
Good catch! Apparently my I confused code and joi when moving them.